extract/omie_extractor.py

```
@@ -0,0 +1,417 @@
+            fecha_fin_carga_dt = datetime.now() - timedelta(days=self.download_window) + timedelta(days=1)
+            
+            fecha_inicio_carga = fecha_inicio_carga_dt.strftime('%Y-%m-%d')
+            fecha_fin_carga = fecha_fin_carga_dt.strftime('%Y-%m-%d')
+            print(f"No se han proporcionado fechas de carga, se descargarán datos entre {fecha_inicio_carga} y {fecha_fin_carga}")
+
+        else:
+            raise ValueError("No se han proporcionado fechas de carga completas")
+
+        return fecha_inicio_carga, fecha_fin_carga
+
+    def extract_omie_diario(self, fecha_inicio_carga: Optional[str] = None, fecha_fin_carga: Optional[str] = None) -> None:
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)